### PR TITLE
jenkins

### PR DESCRIPTION
**Pull request by pierre-emmanuelJ**
Signed-off-by: Pierre-Emmanuel Jacquier <pierre-emmanuel.jacquier@epitech.eu>

### CODE REVIEW

**Comment by mcorbin** [comment]
We could maybe extract some functions like golint or gofmt in the shared lib ?

File: networks.go [hunk 1]
```diff
@@ -247,7 +247,7 @@ func (listNetwork *ListNetworks) SetPageSize(pageSize int) {
 func (*ListNetworks) each(resp interface{}, callback IterateItemFunc) {
 	networks, ok := resp.(*ListNetworksResponse)
 	if !ok {
-		callback(nil, fmt.Errorf("ListNetworksResponse expected, got %t", resp))
+		callback(nil, fmt.Errorf("%q expected, got %t", "ListNetworksResponse", resp))
```
Comment (greut) [inline networks.go]: meh :-)

Comment (greut) [reply networks.go]: `"type error: ListNetworkResponse expected, got %T"`

File: resource_metadata_test.go [hunk 2]
```diff
@@ -13,5 +13,8 @@ func TestListResourceDetailss(t *testing.T) {
 	if req.name() != "listResourceDetails" {
 		t.Errorf("API call doesn't match")
 	}
+	if req.description() != "List resource detail(s)" {
+		t.Errorf("API call doesn't match")
+	}
```
Comment (greut) [inline resource_metadata_test.go]: this is gone if you rebase... at least it should be.

Comment (pierre-emmanuelJ) [reply resource_metadata_test.go]: ok i'll try to rebase

Comment (pierre-emmanuelJ) [reply resource_metadata_test.go]: I have rebased on master but that not resolve this issue

**Comment by greut** [comment]
@pierre-emmanuelJ yay, it's broken again :-)

File: .gometalinter.json [hunk 3]
```diff
@@ -1,4 +1,4 @@
 {
-    "Disable": ["gocyclo", "maligned"],
+    "Disable": ["gocyclo", "goconst", "maligned"],
```
Comment (greut) [inline .gometalinter.json]: @pierre-emmanuelJ

Comment (pierre-emmanuelJ) [reply .gometalinter.json]: Last build: https://ci.internal.exoscale.ch/blue/organizations/jenkins/egoscale/detail/egoscale/457/pipeline


```
cmd/exo/cmd/icmp_code.go:14:2:warning: unused variable or constant netUnreachable (varcheck)

cmd/exo/cmd/icmp_code.go:15:2:warning: unused variable or constant hostUnreachable (varcheck)

cmd/exo/cmd/icmp_code.go:16:2:warning: unused variable or constant protocolUnreachable (varcheck)

cmd/exo/cmd/icmp_code.go:17:2:warning: unused variable or constant portUnreachable (varcheck)

cmd/exo/cmd/icmp_code.go:18:2:warning: unused variable or constant fragmentationNeededAndDoNotFragmentWasSet (varcheck)

cmd/exo/cmd/icmp_code.go:19:2:warning: unused variable or constant sourceRouteFailed (varcheck)

cmd/exo/cmd/icmp_code.go:20:2:warning: unused variable or constant destinationNetworkUnknown (varcheck)

cmd/exo/cmd/icmp_code.go:21:2:warning: unused variable or constant destinationHostUnknown (varcheck)

cmd/exo/cmd/icmp_code.go:22:2:warning: unused variable or constant sourceHostIsolated (varcheck)

```

Comment (pierre-emmanuelJ) [reply .gometalinter.json]: varcheck

**Comment by greut** [comment]
@pierre-emmanuelJ being on :fire: this morning https://www.reddit.com/r/holdmybeer/comments/8vivv3/hmb_got_a_belly_flop_competition_to_win/

**Comment by greut** [comment]
big congrats!!

**Pull request merged**